src/datetimepicker.ios.js

```diff
@@ -80,18 +78,6 @@ export default function Picker({
     [onChange, value],
   );
 
-  React.useEffect(
-    function ensureCorrectHeight() {
```

Comment:
On the old architecture, we definitely don't need this, good catch 👍 

However, I have not tested my PR with the [New Architecture PR](https://github.com/react-native-datetimepicker/datetimepicker/pull/657). I _think_ the shadow view implementation I've done is only for the old architecture given the rendering on the New Architecture is done in C++. But I am not 100 % sure. @alfonsocj, would you be able to check if this library still works on the New Architecture if you'd merge the changes from here?